frontend/src/layout/header/UserBox.tsx

```diff
@@ -282,15 +282,28 @@ const FloatingMenu: React.FC<FloatingMenuProps> = ({ close, type }) => {
     );
 };
 
+// Use this for text elements that exist exclusively for screen reader purposes
+export const visuallyHidden = {
+    clipPath: "inset(50%)",
+    height: 1,
+    width: 1,
+    overflow: "hidden",
+    position: "absolute",
+    whiteSpace: "nowrap",
+} as const;
```

Comment:
This seems to be overly complicated? I assume `display: none` doesn't work as it is also ignored by screen readers? But what about `positon: absolute; visibility: hidden`?

Reply:
`visibility: hidden` also hides the content from screen-readers.
I adapted the above approach from this article: https://webaim.org/techniques/css/invisiblecontent/